sys/netinet/icmp6.h

```diff
@@ -373,6 +374,14 @@ struct nd_opt_dnssl {		/* DNSSL option (RFC 6106) */
 	/* followed by list of DNS search domains */
 } __packed;
 
+struct nd_opt_pref64 {		/* PREF64 option (RFC 8781) */
+	u_int8_t	nd_opt_pref64_type;
+	u_int8_t	nd_opt_pref64_len;
+	/* bits 0-12 are the SL, bits 13-15 are the PLC */
+	u_int16_t	nd_opt_pref64_sl_plc;
```

Comment:
Can you please change to C99 uintXX_t types instead of non-standard BSD 80-ish u_intXX_t.

Reply:
do you want me to change every instance in the file (for consistency) or just the new ones?

Reply:
Just the new ones in the scope of this pull request. However, if you do a sweep across the entire file as a separate change, that would be great.

Reply:
done.